### PR TITLE
Add the Protocol BBEs with Dots in Their Titles to 1.0 & 1.1

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add the protocol BBEs with dots in their titles to 1.0 & 1.1.

> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: 1.0/learn/by-example/http-1-1-to-2-0-protocol-switch.html [hunk 1]
```diff
@@ -2,10 +2,10 @@
 layout: ballerina-example-page
 title: HTTP 1.1 to 2.0 Protocol Switch
 description: Ballerina by Example is a hands-on introduction to Ballerina using annotated example programs. Check out the first example or browse the full list below.
-permalink: /1.1/learn/by-example/http-1.1-to-2.0-protocol-switch
+permalink: /1.0/learn/by-example/http-1-1-to-2-0-protocol-switch
```
Comment (ldclakmal) [inline 1.0/learn/by-example/http-1-1-to-2-0-protocol-switch.html:5]: Do we need to update the ballerina version here?

File: 1.0/learn/by-example/http-1-1-to-2-0-protocol-switch.html [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
 redirect_from:
-  - /v1-1/learn/by-example/http-1.1-to-2.0-protocol-switch
-  - /v1-1/learn/by-example/http-1.1-to-2.0-protocol-switch.html
+  - /v1-0/learn/by-example/http-1-1-to-2-0-protocol-switch
```
Comment (ldclakmal) [inline 1.0/learn/by-example/http-1-1-to-2-0-protocol-switch.html:7]: Do we need to update the ballerina version here also?

File: 1.0/learn/by-example/http-1-1-to-2-0-protocol-switch.html (hunk 1 above)
Comment (praneesha) [reply 1.0/learn/by-example/http-1-1-to-2-0-protocol-switch.html:5]: Fixed

File: 1.0/learn/by-example/http-1-1-to-2-0-protocol-switch.html (hunk 2 above)
Comment (praneesha) [reply 1.0/learn/by-example/http-1-1-to-2-0-protocol-switch.html:7]: Fixed.

**Review by ldclakmal — APPROVED** [review]

**Pull request merged**